test/sensitivities/functional/reduce.jl

```diff
@@ -20,11 +17,11 @@
                     # Test +.
                     x_ = Leaf(Tape(), x)
                     s = functional(f, +, x_)
-                    @test ∇(s)[x_] ≈ ∇.(f, Arg{1}, x)
+                    @test ∇(s)[x_] ≈ derivative_via_frule.(f, x)
                 end
 
                 # Some composite sensitivities.
-                composite_functions = (x->5x, x->1 / (1 + x), x->10+x)
+               composite_functions = (x->5x, x->1 / (1 + x), x->10+x)
```

Comment:
This whitespace change seems to be an error

Reply:
good catch
```suggestion
                composite_functions = (x->5x, x->1 / (1 + x), x->10+x)
```